zh_CN/LC_MESSAGES/django.po

```diff
@@ -7757,9 +7757,11 @@ msgid ""
 "                This value may contain only letters, numbers, and @/./+/-/_ characters.\n"
 "                "
 msgstr ""
-"请注意用户名将公开可见。\n"
-"这是必填字段，不能与他人重复，且须不超过 150 个字符。\n"
-"字段内容仅可包含拉丁字母、数字、@ . + - _ 等符号。"
+"\n"
+"                请注意用户名将公开可见。\n"
+"                这是必填字段，不能与他人重复，且须不超过 150 个字符。\n"
+"                字段内容仅可包含拉丁字母、数字、@ . + - _ 等符号."
```

Comment:
I think we should keep the same period-like character that was used in the translation.
```
"                字段内容仅可包含拉丁字母、数字、@ . + - _ 等符号。"
```